index.d.ts

```diff
@@ -582,6 +584,36 @@ export declare interface User {
   [key: string]: string | undefined
 }
 
+export declare interface Appsec {
+  /**
+   * Links success login related data to the current trace, it calls to setUser internally
+   * @param {User} user Properties of the authenticated user. Accepts custom fields.
+   * @param {[key: string]: string} metadata Custom fields to login success event
+   *
+   * @beta This method is in beta and could change in future versions
+   */
+  trackUserLoginSuccessEvent(user: User, metadata?: { [key: string]: string }): void
+
+  /**
+   * Links login failure related data to the current trace
+   * @param {string} userId The id of the user who has tried to login
+   * @param {boolean} exists If the user id exists
+   * @param {[key: string]: string} metadata Custom fields to login failure event
+   *
+   * @beta This method is in beta and could change in future versions
+   */
+  trackUserLoginFailureEvent(userId: string, exists: boolean, metadata?: { [key: string]: string }): void
+
+  /**
+   * Links custom data to the current trace
+   * @param {string} eventName The name of the custom event
+   * @param {[key: string]: string} metadata Custom fields to login custom event
```

Comment:
`Custom fields to login custom event` ?

Reply:
ups, it isn't "login custom event", it is only "custom event"